torch/_inductor/codegen/common.py

```diff
@@ -1717,7 +1717,14 @@ def rename_indexing(self, index) -> sympy.Expr:
         replacements = {
             x: self.args.size(x)
             for x in sorted_symbols
-            if symbol_is_type(x, (SymT.UNBACKED_INT, SymT.SIZE, SymT.PRECOMPUTED_SIZE))
+            if symbol_is_type(
+                x,
+                (
+                    SymT.UNBACKED_INT,
+                    SymT.SIZE,
+                    SymT.PRECOMPUTED_SIZE,
+                ),
+            )
```

Comment:
?

Reply:
oops, I was fiddling around with adding FLOAT to the test here, I eventually decided to get rid of it but the formatting change stuck lol